provider/resource_overrides.rb

```diff
@@ -12,22 +12,22 @@
 # limitations under the License.
 
 require 'api/object'
+require 'provider/property_override'
 require 'provider/resource_override'
 
 module Provider
-  # A hash of Provider::Override objects where the key is the api name for that
-  # object.
+  # A hash of Provider::ResourceOverride objects where the key is the api name
+  # for that object.
   #
   # Example usage in a provider.yaml file where you want to extend a resource
   # description:
   #
   # overrides: !ruby/object:Provider::ResourceOverrides
-  #   SomeResource:
-  #     !ruby/object:Provider::MyProvider::Override
-  #     description: |
-  #       {{ description}}
-  #
-  #       A tool-specific description complement
+  #   SomeResource: !ruby/object:Provider::MyProvider::ResourceOverride
+  #     description: '{{description}} A tool-specific description complement'
+  #     properties:
+  #       someProperty: !ruby/object:Provider::MyProvider::PropertyOverride
```

Comment:
Let's also talk about how to get nested object overrides. Please add an example here.

Reply:
Done